pkg/testcase/certrotate.go

```diff
@@ -0,0 +1,109 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCertRotate() {
+	// Please refer to docs/testing.md for instructions on setup for this test
+
+	cluster := factory.AddCluster(GinkgoT())
+	serverIPs := cluster.ServerIPs
+	agentIPs := cluster.AgentIPs
+	p, _ := shared.GetProduct()
```

Comment:
you shoulnd never omit the err var, just in very rare occasions

Reply:
fixed